In [6]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns
import pickle
from matplotlib import font_manager, rc
font_name = font_manager.FontProperties(fname="c:/Windows/Fonts/malgun.ttf").get_name()
rc('font', family=font_name)

In [7]:
%matplotlib inline

In [37]:
inchoen_move.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 10 entries, 5 to 59
Data columns (total 3 columns):
 #   Column  Non-Null Count  Dtype 
---  ------  --------------  ----- 
 0   지역      10 non-null     object
 1   구분      10 non-null     object
 2   2020    10 non-null     int32 
dtypes: int32(1), object(2)
memory usage: 280.0+ bytes


In [105]:
inchoen_move = pd.read_csv("인천_move.csv")
inchoen_move = inchoen_move[(inchoen_move['구분']=='외부방문자 전년대비증감률(%)')]
inchoen_move['2020'] = inchoen_move['2020'].astype(int)
inchoen_move[inchoen_move['2020']>0]

지역                구분  2020
53  강화군  외부방문자 전년대비증감률(%)     1
59  옹진군  외부방문자 전년대비증감률(%)     7

In [109]:
Region = ["강원_move.csv","경기_move.csv","경남_move.csv","경북_move.csv","광주_move.csv","대구_move.csv","대전_move.csv","부산_move.csv",
         "서울_move.csv","세종_move.csv","울산_move.csv","인천_move.csv","전남_move.csv","전북_move.csv","제주_move.csv","충남_move.csv",
         "충북_move.csv"]
result = []
for i in Region :
    move = pd.read_csv(i)
    move = move[(move['구분']=='외부방문자 전년대비증감률(%)')]
    move['2020'] = move['2020'].astype(int)
    print(i)
    print(move[move['2020']>0][['지역','2020']])

강원_move.csv
      지역  2020
101  고성군     4
107  양양군    10
경기_move.csv
       지역  2020
125   구리시     4
131  남양주시     2
161   하남시     6
197   안성시     3
227   포천시     1
239   연천군     5
245   가평군     3
251   양평군     1
경남_move.csv
     지역  2020
59  밀양시     7
77  의령군     1
95  고성군     1
경북_move.csv
      지역  2020
107  고령군     5
광주_move.csv
Empty DataFrame
Columns: [지역, 2020]
Index: []
대구_move.csv
Empty DataFrame
Columns: [지역, 2020]
Index: []
대전_move.csv
Empty DataFrame
Columns: [지역, 2020]
Index: []
부산_move.csv
     지역  2020
47   북구     5
95  기장군     5
서울_move.csv
Empty DataFrame
Columns: [지역, 2020]
Index: []
세종_move.csv
Empty DataFrame
Columns: [지역, 2020]
Index: []
울산_move.csv
Empty DataFrame
Columns: [지역, 2020]
Index: []
인천_move.csv
     지역  2020
53  강화군     1
59  옹진군     7
전남_move.csv
      지역  2020
53   고흥군     6
107  영광군     1
125  진도군     1
전북_move.csv
     지역  2020
41  김제시     1
제주_move.csv
Empty DataFrame
Columns: [지역, 2020]
Index: []
충남_move.csv
     지역  2020
53  당진시     1
95  태안군    

강원_move.csv
101  고성군     4
107  양양군    10
경기_move.csv
125   구리시     4
131  남양주시     2
161   하남시     6
197   안성시     3
227   포천시     1
239   연천군     5
245   가평군     3
251   양평군     1
경남_move.csv
59  밀양시     7
77  의령군     1
95  고성군     1
경북_move.csv
107  고령군     5
부산_move.csv
47   북구     5
95  기장군     5
인천_move.csv
53  강화군     1
59  옹진군     7
전남_move.csv
53   고흥군     6
107  영광군     1
125  진도군     1
전북_move.csv
41  김제시     1
충남_move.csv
53  당진시     1
95  태안군     2

In [90]:
result.append(move[move['2020']>0])
result[1]
result[2]
result[3]
result[7]
result[11]
result[12]
result[13]
result[15]

지역                구분  2020
53  당진시  외부방문자 전년대비증감률(%)     1
95  태안군  외부방문자 전년대비증감률(%)     2

지역                구분  2020
53  강화군  외부방문자 전년대비증감률(%)     1
59  옹진군  외부방문자 전년대비증감률(%)     7